samples/SkiaJsSample/src/wasmJsMain/resources/index.html

```diff
@@ -9,10 +9,10 @@
 
 <body>
 <h1>Skiko running with K/Wasm</h1>
-<canvas id="c1" width="300" height="300"></canvas>
-<canvas id="c2" width="300" height="300"></canvas>
+<canvas id="c1" width="600" height="600" style="width: 300px; height: 300px"></canvas>
+<canvas id="c2" width="600" height="600" style="width: 300px; height: 300px"></canvas>
 <div>
-    <canvas id="c3" width="606" height="400"></canvas>
+    <canvas id="c3" width="1212" height="800" style="width: 606px; height: 400px"></canvas>
```

Comment:
is width and heighr are explicitly hardcoded here to 2x becasue we usually run the example on Mac with retina display? :) 
So it won't work well on a usual display?

Reply:
a valid note, let me introduce some helper function to the samples.